### ISSUE TITLE
[Bug] Preview cache not cleaned up?

### ISSUE THREAD

**Issue by Ian2020** [previm/previm#159]
I found my preview cache had become enormous after many months use. I can see some code `PrevimCleanup` that looks like it wants to remove previews on the VimLeave event:

```vim
exe printf("au VimLeave * call previm#cleanup_preview('%s')", dir)
```

But looks like it was wrapped in printf some years ago. In additional it seems Previm will generate previews whether `:PrevimOpen` has been run or not, creating previews for files that may never be needed by the user.

**Comment by mattn** [comment]
Thanks for the report. Now I don't have enoguh time to fix this. Can you fix this?

**Comment by mattn** [comment]
Closed by #160
